core-bundle/src/Resources/contao/library/Contao/Database/Result.php

```diff
@@ -363,7 +366,13 @@ public function count()
 	{
 		if ($this->rowCount === null)
 		{
-			$this->rowCount = $this->resResult->rowCount();
+			if (
+				$this->resResult instanceof DoctrineStatement
+				|| $this->resResult instanceof DoctrineResult
```

Comment:
`Doctrine\DBAL\Result` only exists in doctrine/dbal 3.0, so I assume this is a forward-compatibility layer, right? Still I wonder how this condition can ever be `true` given the if statement in line 85.

Reply:
Exists in 2.13.0 too https://github.com/doctrine/dbal/blob/2.13.x/lib/Doctrine/DBAL/Result.php

Reply:
Ah, new file structure in version 3. 👍🏻